features/buyer/catalogue.feature

```diff
@@ -73,3 +73,12 @@ Scenario: User is able to search by keywords field on the search results page to
   Then I see that service.id in the search summary text
   And I see that service.id as the value of the 'q' field
   And I see that service in the search results
+
+Scenario: User is able to click a filter on the search results page to narrow down the results returned
+  Given I am on the /g-cloud page
+  And I have a random g-cloud service from the API
```

Comment:
I think it boils down to this:

* we currently have no way to identify a service that should be in the live index, based on any data stored in the API
* which frameworks are currently indexed depends only on the calls made (manually) to the indexing script.

Feels like the only way to fix that would be to introduce a new 'indexed' flag on the frameworks table, and have everything hang off that.

Alternatively we could introduce a new status, i.e. "live-but-not-indexed" (with a pithier name) - but we'd need to review all code currently checking for status "live", so that seems riskier.